client/src/services/socket.service.js

```diff
@@ -0,0 +1,26 @@
+import io from 'socket.io-client';
+import { app } from '../app.js'
+
+export class SocketService {
```

Comment:
why don't we export this service as singleton?
I don't think we need to create a new instance for every connection.
let's do
`export const socketService = new SocketService()`